Server/src/Server.Application/ViewModels/BookViewModel.cs

```diff
@@ -5,5 +5,13 @@ namespace Server.Application.ViewModels;
 public class BookViewModel
 {
     public Guid Id { get; set; }
-    public string Name { get; set; }
+    public string ISBN { get; set; }
+    public string Title { get; set; }
+    public int YearPublished { get; set; }
+    public string? CoverLink { get; set; }
+    public double? AverageRating { get; set; }
+    public double? AverageCriticRating { get; set; }
+    public PublisherViewModel Publisher { get; set; }
+    public ICollection<AuthorViewModel> Authors { get; set; }
+    public ICollection<BookCategoryViewModel> BookCategories { get; set; }
```

Comment:
I teraz pytanie, jaką przyjmujemy taktykę, bo powiedzmy np `BookCategories` nie będzie nam potrzebne (ranking czy jakieś podsumowanie rezerwacji) to wtedy wstawiamy po prostu tutaj pustą listę czy robimy te propertisy nullable?

Reply:
Ale fakt, że chociażby do widoku listy książek a widoku danej książki będą potrzebne inne dane, także to trzeba ustalić dosyć szybko